rgsync/Connectors/sql_connectors.py

```diff
@@ -174,23 +192,6 @@ def WriteData(self, data):
             WriteBehindLog(msg)
             raise Exception(msg) from None
 
-class MySqlConnector(BaseSqlConnector):
```

Comment:
Why deleting the MySql connector?

Reply:
That was a mistake and Ii can blame my glasses. Let me fix it